### PR TITLE
feat: add info banner to admin form builder when webhooks are enabled

### LINKED ISSUE #3954: [React] Add warning when webhooks are enabled

### PR DESCRIPTION
**Pull request by karrui**
## Problem

Closes #3954

### CODE REVIEW

**Review by wanlingt — APPROVED** [review]
lgtm!

**Comment by karrui** [comment]
will wait for #4525 to go in first before I reopen this... if not rebase hell again

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FormBuilder.tsx [hunk 1]
```diff
@@ -53,95 +56,108 @@ export const FormBuilder = ({
...
-        h="fit-content"
-        spacing={{ base: 0, md: '1.5rem' }}
-        bg={bg}
+    <Box w="100%">
+      {settings?.webhook?.url ? (
+        <Banner showCloseButton={false}>
+          Webhooks are enabled on this form. Please ensure the webhook server is
+          able to handle any field changes.
+        </Banner>
+      ) : null}
```
Comment (karrui) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FormBuilder.tsx]: added new box and the banner here. all other layout stayed the same

**Review by wanlingt — APPROVED** [review]
looks good!

**Pull request merged**